src/components/views/voip/GroupCallView.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useCallback } from "react";
+import { GroupCall } from "matrix-js-sdk/src/webrtc/groupCall";
+import { useGroupCall } from "./GroupCallView/useGroupCall";
+import { VideoGrid } from "./GroupCallView/VideoGrid";
+
+interface IProps {
+    groupCall: GroupCall;
+    pipMode: boolean;
+}
+
+function useRoomLayout(): [string, () => void] {
+    const [layout, setLayout] = useState("gallery");
+
+    const toggleLayout = useCallback(() => {
+        setLayout(layout === "spotlight" ? "gallery" : "spotlight");
+    }, [layout]);
+
+    return [layout, toggleLayout];
+}
+
+export function GroupCallView({ groupCall, pipMode }: IProps) {
```

Comment:
```suggestion
const GroupCallView: React.FC<IProps> = ({ groupCall, pipMode }) => {
```

We tend to this rather than the other thing, you'll also need to export by default because of the dumb way `component.index` works